crabml-core/src/backends/cpu/buf/buf_q8_0.rs

```diff
@@ -292,12 +292,12 @@ mod impl_x86_64_avx2 {
     }
 
     pub fn vec_dot_q8_0_q8_0(abs: &[BlockQ8_0], bbs: &[BlockQ8_0]) -> f32 {
-        assert_eq!(
-            bbs.len() % 2,
-            0,
-            "bbs.len() must be a multiple of 64, got: {}",
-            bbs.len()
-        );
+        // assert_eq!(
```

Comment:
yesterday I learned about `debug_assert_xx` from @MrCroxx, it looks useful that it could skip the checking in the release build qwq

Reply:
> yesterday I learned about debug_assert_xx from @MrCroxx, it looks useful that it could skip the checking in the release build qwq
BTW `debug_assert_xx` is supposed to be used when there is enough unit tests that can make sure all unexcepted cases can be found in test with debug mode and don't want the assertion affect the performance in release mode. IMO, it is not suitable here.